tests/data/example-def-with-lists.yaml

```diff
@@ -0,0 +1,84 @@
+apiVersion: v1
+kind: SecretList
+items:
+- apiVersion: v1
+  kind: Secret
+  metadata:
+    name: "nginxsecret"
+    namespace: "default"
+  type: kubernetes.io/tls
+  data:
+    tls.crt: "LS0tLS1CRUdJTiBDRVJUSUZJQ0FURS0tLS0tCk1JSURIekNDQWdW...Ci0tLS0tRU5EIENFUlRJRklDQVRFLS0t"
+    tls.key: "LS0tLS1CRUdJTiBQUklWQVRFIEtFWS0tLS0tCk1JSUV2UUlCQURB...TkJnVW1VbGc9Ci0tLS0tRU5EIFBSSVZB"
+---
+apiVersion: v1
+kind: List
```

Comment:
Do you mean `MydbList`?

Reply:
I'm not sure that spec exists in apiVersion: v1.  I was under the impression that `Mydb` is a CRD.  It's not likely there's a definition of `MydbList`


according to the Note on [kubernetes collections](https://kubernetes.io/docs/reference/using-api/api-concepts/#collections) 

> Keep in mind that the Kubernetes API does not have a kind named List.

> kind: List is a client-side, internal implementation detail for processing collections that might be of different kinds of object. Avoid depending on kind: List in automation or other code.

Reply:
I see, so if I query the API endpoint for this CRD directly I get `kind: MydbList` but kubectl is dumping `kind: List` instead. And for this feature you expect people to create definition files following kubectl format, correct?

Reply:
Honestly, the situation I'm trying to address in this feature is where I have yaml definitions like [this one](https://github.com/kubernetes/cloud-provider-openstack/blob/v1.27.1/manifests/controller-manager/cloud-controller-manager-roles.yaml) 

in this case it's a `v1` `List` of `ClusterRoles`.  I'd like lightkube to be able to read this file exactly as it's produced here in this repo.

i've gone back and forth reading the kubernetes docs, and looking at API queries and for the life of me I can't determine if a good test is `kind: MydbList` or `kind: List`

i *THINK* if you use kubectl to get `kubectl get mydbs -o yaml` it would return `kind: List` -- just l ike it does for `kubectl get clusterroles -o yaml`

these internal `.*List` types are converted into simply `List`

in either case, i think lightkube should handle loading any kind where the value ends with `List` and there's an `items` list attached

Reply:
Agree, let's keep it like this.